wireapp/wire-android
# chore: use manifest placeholder to get rid of playstore error (WPB-8645)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8645" title="WPB-8645" target="_blank"><img alt="Epic" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10807?size=medium" />WPB-8645</a>  [Android] Infrastructure code and developer experience
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We have an "error/warning" from the play store console, suggesting to not use keys directly
Generated `BuildConfig` are plain text and accessible by the scanner of the playstore console.

### Causes (Optional)

We are seeing that we can not release Beta/Blue apps in the playstore to internal testers


### Solutions

Follow "best practices" and use manifest placeholder instead for this API key.

### How to test / testing

The app should build, and if you build locally, you should see the substitution in the merged `AndroidManifest` from the build directory.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
